src/Rule/Compare.php

```diff
@@ -15,6 +15,20 @@
 use Yiisoft\Validator\WhenInterface;
 
 /**
+ * Abstract base for all the comparison validation options.
+ *
+ * The value being compared with {@see Compare::$targetValue} or {@see Compare::$targetAttribute}, which is set
+ * in the constructor.
+ *
+ * It supports different comparison operators, specified
```

Comment:
Suggest to list them with equivalent code. For example:

- {@see Equal} is a shortcut for `new CompareTo(operator: '===')`.

and so on.

Also mention `CompareTo`.